x/paramHub/keeper/keeper.go

```diff
@@ -130,6 +134,28 @@ func (keeper *Keeper) EndBreatheBlock(ctx sdk.Context) {
 	return
 }
 
+func (keeper *Keeper) SetSyncFeeAfterFirstSunsetFork(ctx sdk.Context) {
+	log := keeper.Logger(ctx)
+	if sdk.IsUpgrade(sdk.FirstSunsetFork) && keeper.ScKeeper != nil {
```

Comment:
Is `sdk.IsUpgrade(sdk.FirstSunsetFork)` required? because it is only called only once at the hardfork of `FirstSunsetFork`